lib/loader/serialize/serial_type.cpp

```diff
@@ -86,6 +117,15 @@ Serializer::serializeValType(const ValType &Type, ASTNodeAttr From,
   case TypeCode::Ref:
   case TypeCode::RefNull:
     return serializeRefType(Type, From, OutVec);
+  case TypeCode::NullFuncRef:

```

Comment:
Not need the following cases.
Because in the `class ValType`, we migrated the ref-related cases into the `ref` or `ref.null`, and handled in the heap type.

Reply:
Not fixed.